.github/workflows/check-pr-title.yaml

```diff
@@ -0,0 +1,38 @@
+name: Check PR Title
+
+on:
+  pull_request_target:
+    types:
+      - edited
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  check:
+    name: Check PR Title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@v5.5.3
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          types: |
+            chore
+            fix
+            feat
+            revert
+
+          scopes: |
+            api
+            ci
+            controller
+            deps
+            docs
+            initializer
+            manifests
+            plugins
+            runtimes
+            test
+            webhook
```

Comment:
Why do we need to split the trainer-controller-manager into controller, plugins, and webhook?
Isn't a single `controlplane` or `manager` enough?

Reply:
I agree this might be too fine grained if the goal is to organize change log, and separating control plane from runtime would be sufficient enough.

Maybe `operator` could also be a viable alternative to cover controller, plugins, webhook.

I wonder also whether `initializer` could be included into `runtimes`.

Reply:
The `operator` sounds reasonable since that is the widely-used term defined by the OperatorFramework.

Reply:
Sounds good!

> I wonder also whether initializer could be included into runtimes.

Actually, runtimes and initializers are two separate components for Kubeflow Trainer, isn't ?
I expect that initializer changes just go here: https://github.com/kubeflow/trainer/tree/master/pkg/initializers